### PR TITLE
fix(FX-3559): Dropdown filters on /galleries page

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Fix**

This PR resolves [FX-3559]

### Description
* As a user
* When I go to the `/galleries` page
* And specify some filter
* I see that nothing is happening

### Some context
* [Slack thread](https://artsy.slack.com/archives/C9SATFLUU/p1636452319117100)
* PR after which the problem appeared artsy/force#8707

#### Before
https://user-images.githubusercontent.com/3513494/140926689-1c6efbf9-b02a-412f-a6a6-f0eb2c9547c1.mp4

#### After
https://user-images.githubusercontent.com/3513494/140926751-f5ab5e7b-ee92-45f8-974b-24fd133be92f.mp4

[FX-3559]: https://artsyproduct.atlassian.net/browse/FX-3559?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee [hunk 1]
```diff
@@ -1,8 +1,8 @@
 _ = require 'underscore'
 Profile = require '../../../../models/profile.coffee'
+FilterPartners = require './collections/filter_partners.coffee'
```
Comment (gkartalis) [inline src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee:3]: 😬

**Review by gkartalis — APPROVED** [review]
Looking great! Thanks for the fix!

File: src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee (hunk 1 above)
Comment (dimatretyak) [reply src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee:3]: @mc-jones should we refactor this and rewrite it from `CoffeeScript` to `JS`?

Comment (damassi) [reply src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee:3]: This page is nearly done being rebuilt so no need to convert; we'll soon be deleting

**Pull request merged**